connector/rest/src/main/java/it/pagopa/selfcare/external_api/connector/rest/mapper/InstitutionMapper.java

```diff
@@ -91,10 +91,10 @@ default OffsetDateTime map(LocalDateTime localDateTime) {
                 .orElse(null);
     }
 
-    @Mapping(target = "onboarding", source = "onboarding", qualifiedByName = "convertOnboading")
+    @Mapping(target = "onboarding", source = "onboarding", qualifiedByName = "convertOnboarding")
     Institution toInstitution(it.pagopa.selfcare.core.generated.openapi.v1.dto.InstitutionResponse body);
 
-    @Named("convertOnboading")
+    @Named("convertOnboarding")
```

Comment:
it might be better "toOnboarding"

Reply:
There is other method named toOnboarding

Reply:
you could add the difference...for ex: toOnboardingWithDate